eth/backend.go

```diff
@@ -44,6 +44,8 @@ type Backend interface {
 	ChainID(ctx context.Context) (*big.Int, error)
 	MaxGasPrice() *big.Int
 	SetMaxGasPrice(gp *big.Int)
+	MinGasPrice() *big.Int
+	SetMinGasPrice(gp *big.Int)
```

Comment:
In https://github.com/livepeer/go-livepeer/pull/1923/files `MaxGasPrice` and `SetMaxGasPrice` are moved to the `GasPriceMonitor` 

Instead backend now exposes `GasPriceMonitor() *GasPriceMonitor`  to give the CLI server access to the `GasPriceMonitor` by calling `node.Eth.Backend().GasPriceMonitor()` for setting and getting the max gas price through the CLI. 

I'm not sure if that makes sense to do but I found it a bit cleaner than having the setters/getters for a particular service (i.e. GasPriceMonitor) live on a different API than the actual service. WDYT of doing that here as well so that here as well ?

Reply:
As discussed offline, the plan is to merge this PR as-is and address the GasPriceMonitor related changes mentioned in this comment in https://github.com/livepeer/go-livepeer/pull/1923 since that PR refactors the `backend` type in the `eth` package to expose a `GasPriceMonitor()` method.